projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -31,32 +35,74 @@ public ProjectCreationService(ProjectRepository projectRepository) {
    */
   public Result<Project, ApplicationException> createProject(String title, String objective) {
     ProjectTitle projectTitle;
+    try {
+      projectTitle = ProjectTitle.create(title);
+    } catch (RuntimeException e) {
+      log.error(e.getMessage(), e);
+      return Result.failure(new ProjectManagementException(ErrorCode.INVALID_PROJECT_TITLE,
+          ErrorParameters.of(200, title)));
```

Comment:
missing the exception handling for other Throwables here

Reply:
I think all throwables are not handled. Checked exceptions are not handled as they should not occur and are truly exceptional.
Maybe I misunderstood you here?